src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/controller/sw/SwitchFsm.java

```diff
@@ -370,6 +367,13 @@ private void updateFeatures(Set<SwitchFeature> update) {
         }
     }
 
+    private AbstractPort addNewPort(SwitchFsmContext context) {
```

Comment:
I wold like to not use context here, use `Edpoint` instance or even just `portNumber` should be enough.